deployments/rialto/bridge-config/poa-exchange-tx-generator-entrypoint.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```

Comment:
Can you add some documentation as to what the goal of the script is? Some things I think are worth noting are that it's not meant from production use, and it more for lightly "fuzzing" a test network